src/System.Runtime.Numerics/ref/System.Runtime.Numerics.cs

```diff
@@ -246,5 +248,8 @@ public partial struct Complex : System.IEquatable<System.Numerics.Complex>, Syst
         public string ToString(System.IFormatProvider provider) { throw null; }
         public string ToString(string format) { throw null; }
         public string ToString(string format, System.IFormatProvider provider) { throw null; }
+        public static bool IsFinite(System.Numerics.Complex value) { throw null; }
```

Comment:
nit: It would be nice if these were ordered alphabetically and were grouped with the other `static methods` in the file.

Reply:
(same for the implementation file)